src/components/ResortsList.js

```diff
@@ -11,4 +12,12 @@ class ResortsList extends Component{
     )
   }
 }
-export default ResortsList;
\ No newline at end of file
+
+ResortsList.PropTypes={
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  city: PropTypes.string.isRequired
+}
```

Comment:
I was wondering what sholud be in ResortsList.PropTypes - on the one hand we need to check type of  id, name and city but on the other hand resort is porps of ResortsList component, so i should change it like this: 
ResortsList.propTypes={
  resort: PropTypes.object
}
Am I right?

Reply:
You can check nested props of the object with `PropTypes.shape`
https://reactjs.org/docs/typechecking-with-proptypes.html

BTW this component should be renamed as it handles rendering of only one resort